types/rn-randomnumber/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for rn-randomnumber 1.0
+// Project: https://www.npmjs.com/package/rn-randomnumber
+// Definitions by: Rajas Paranjpe <https://github.com/ChocolateLoverRaj>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function randomNoGenerator(min: number, max: number): number;
```

Comment:
```suggestion
declare function randomNoGenerator(min?: number, max?: number): number;
```
it support quick and dirty usage:
```ts
> rn()
0.7837907906151589
> rn()
0.61723048281547
> rn()
0.650512321255567
```